lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -260,41 +261,12 @@ def create(work_package, context = {})
                  exec_context: :decorator,
                  getter: -> (*) { datetime_formatter.format_datetime(represented.updated_at) }
 
-        property :status,
-                 embedded: true,
-                 class: ::Status,
-                 decorator: ::API::V3::Statuses::StatusRepresenter
-        property :author,
-                 embedded: true,
-                 class: ::User,
-                 decorator: ::API::V3::Users::UserRepresenter
-        property :responsible,
-                 embedded: true,
-                 class: ::User,
-                 decorator: ::API::V3::Users::UserRepresenter
-        property :assigned_to,
-                 as: :assignee,
-                 embedded: true,
-                 class: ::User,
-                 decorator: ::API::V3::Users::UserRepresenter
-        property :category,
-                 embedded: true,
-                 class: ::Category,
-                 decorator: ::API::V3::Categories::CategoryRepresenter
-        property :priority,
-                 embedded: true,
-                 class: ::IssuePriority,
-                 decorator: ::API::V3::Priorities::PriorityRepresenter
-
```

Comment:
@NobodysNightmare and another strike by the super hero his enemies call "THE METAPROGRAMMER"

![fmn5u 1](https://cloud.githubusercontent.com/assets/617519/6593281/8703a054-c7d7-11e4-8abc-0a124ab4ffe1.gif)